strings/strings.json

```diff
@@ -7369,6 +7369,46 @@
     "en": "Remove filter on {0}",
     "fr": "Enlever le filtre sur {0}"
   },
+  "Select": {
+    "en": "Select",
+    "fr": "Sélectionnez"
+  },
+  "Unselect": {
+    "en": "Unselect",
+    "fr": "Désélectionnez"
+  },
+  "With": {
+    "en": "With",
+    "fr": "Avec"
+  },
+  "ExpandFacetValues": {
+    "en": "Expand facet values",
+    "fr": "Étendre les valeurs de facette"
+  },
+  "CollapseFacetValues": {
+    "en": "Collapse facet values",
+    "fr": "Réduire les valeurs de facette"
+  },
+  "FacetSearch": {
+    "en": "Facet search",
+    "fr": "Recherche de facette"
+  },
+  "PressKeyToActivateFacetSearch": {
```

Comment:
Is this really needed ? Just the keyword `Search` should work, I don't think it's a good idea to put an explanation like that in the tooltip.

If we need something to tell screen readers that an action is possible using they keyboard on a specific control, there are dedicated "aria" attributes that we can use for this.

Reply:
Okay, I will remove this from the title attribute on the facet search.
The facet search is marked as a button giving the screen reader a hint that it is something that can be pressed.
When pressed, the reader says "search edit". Hopefully this is enough indication of what is happening for now.